products/cdk/get-started.adoc

```diff
@@ -90,7 +90,7 @@ To provide `ssh` and `rsync` on Microsoft Windows, install http://cygwin.com/ins
 * http://cygwin.com/setup-x86_64.exe[setup-x86_64.exe, window='_blank'] for 64-bit versions of Windows.
 
 
-### Download the Container Runtime Environment
+### Download the Container Development Environment
```

Comment:
Not AFAIK. See https://docs.google.com/document/d/17hbzJY2eV4dwjKO2p_4mOh3Xqno9LgUobOnTHmp4iMo/edit